kubernetes_deploy/staging/queue_worker_deployment.yml

```diff
@@ -5,7 +5,7 @@ metadata:
     kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
   name: laa-legal-adviser-api-worker
 spec:
-  replicas: 1
+  replicas: 4
```

Comment:
Could you explain in the commit on why this change was made, please? During standup, you mentioned that in-pod concurrency was not making much of a difference, which would be valuable context here.

What is the time to upload how many records with one and with four replicas? It would be great to establish our desired target time to load a specific test spreadsheet.

Reply:
Additionally, we default the `WORKER_APP_CONCURRENCY` setting to 2, which may be way too low. We could increase that to 4, 8, 16, etc., balancing between resource use and pods.

Reply:
I Reset the `replicas` to 1 and increased WORKER_APP_CONCURRENCY to 8 to avoid over provisioning. Hopefully when production starts using node_type of `cache.t2.medium` the performance figures in the PR body will improve.

Reply:
I'm confused, it still says 4 in cf928ce